diff --git a/sphinx_mdinclude/sphinx.py b/sphinx_mdinclude/sphinx.py
index 44e0f87..d006feb 100644
--- a/sphinx_mdinclude/sphinx.py
+++ b/sphinx_mdinclude/sphinx.py
@@ -6,10 +6,9 @@ import os
 import os.path
 
 from docutils import io, nodes, statemachine, utils
-from docutils.core import ErrorString
+from docutils.utils.error_reporting import ErrorString, SafeString
 from docutils.parsers import rst
 from docutils.parsers.rst import directives as rst_directives
-from docutils.utils import SafeString
 
 from . import RestMarkdown
 
